way_to_home/tests/unittests/place/testview.py

```diff
@@ -0,0 +1,245 @@
+"""
+Place view tests
+===============
+"""
+import json
+from django.urls import reverse
+
+from custom_user.models import CustomUser
+from django.test import TestCase, Client
+
+from place.models import Place
+
+
+class PlaceViewTest(TestCase):
+    """TestCase for providing Place view testing"""
+
+    def setUp(self):
+        """Method that provides preparation before testing Place view`s features."""
+        custom_user = CustomUser.objects.create(id=2, email='mymail@icloud.com', is_active=True)
+        custom_user.set_password('qwerty12345')
+        custom_user.save()
+
+        self.client = Client()
+        login = self.client.login(email='mymail@icloud.com', password='qwerty12345')
+        self.assertTrue(login)
```

Comment:
The same situation as mentioned in the model's test.

Reply:
Also removed